lib/views/base/screen_controller_base.dart

```diff
@@ -1,9 +1,10 @@
 import 'package:meta/meta.dart';
+import 'package:momento_booth/utils/logger.dart';
 import 'package:momento_booth/views/base/build_context_abstractor.dart';
 import 'package:momento_booth/views/base/build_context_accessor.dart';
 import 'package:momento_booth/views/base/screen_view_model_base.dart';
 
-abstract class ScreenControllerBase<T extends ScreenViewModelBase> with BuildContextAbstractor {
+abstract class ScreenControllerBase<T extends ScreenViewModelBase> with BuildContextAbstractor, Logger {
```

Comment:
You put it in the base so you don't need to put it in every screen individually?